### PR TITLE
218 Localized UI texts (Italian, Spanish etc) are not displayed

### PR DESCRIPTION
**Pull request by maurizioandreotti**
This PR does:
- fix UI interface traslations
- update localized messages for Italian and Spanish translation
- update about window adding logo

### CODE REVIEW

File: d_rats/version.py [hunk 1]
```diff
@@ -54,12 +54,12 @@
 DRATS_NAME = "d-rats"
 DRATS_DESCRIPTION = "D-RATS"
 DRATS_LONG_DESCRIPTION = _("A communications tool for D-STAR")
-AUTHORS = "Dan Smith, KK7DS" \
-          " Maurizio Andreotti, IZ2LXI" \
-          " Marius Petrescu, YO2LOJ" \
-          " John E. Malmberg, WB8TYW"
+AUTHORS = "Dan Smith, KK7DS \n" \
```
Comment (wb8tyw) [inline d_rats/version.py:57]: Just a note, have to make sure these tests do not break assumptions in the python packaging script that also use them.

File: ui/mainwindow.glade
Comment (wb8tyw) [inline ui/mainwindow.glade:1]: Why is this file empty?

File: ui/mainwindow.glade.h
Comment (wb8tyw) [inline ui/mainwindow.glade.h:1]: Hmm, if this is needed, for apparently we have not been keeping it up to date with the ui.mainwindow.glade file.
Maybe we are missing a step in building the pot.

The POT file either must be reading the.glade file or the .glade.h file.

**Review by wb8tyw — CHANGES_REQUESTED** [review]
Looks like a start, and there is a lot we need to understand, such as why the .glade file is being removed.

**Comment by wb8tyw** [comment]
I will try to find some time next week to so some testing of this PR.

File: d_rats/mainapp.py [hunk 2]
```diff
@@ -1027,6 +1029,18 @@ def _refresh_lang(self):
             gettext.install("D-RATS")
             _ = gettext.gettext
 
+        except Exception as error: # pylint: disable=broad-except
```
[the remark this answers is not in the document]
Comment (maurizioandreotti) [reply d_rats/mainapp.py:1032]: as you know i am not a programmer, so lack of experience. so i would need some advice here

in my local machine the code was working without needing these lines, but when i put it on github it started complaining and blocking the pr, so i had to add these exceptions to have the PR done

File: ui/mainwindow.glade
Comment (maurizioandreotti) [reply ui/mainwindow.glade:1]: I see it with contents... maybe I did some errors in creating the Pr?

File: d_rats/version.py [hunk 3]
```diff
@@ -54,12 +54,12 @@
...
+          "Maurizio Andreotti, IZ2LXI \n" \
+          "Marius Petrescu, YO2LOJ \n" \
+          "John E. Malmberg, WB8TYW \n\n"
 AUTHORS_EMAIL = "Dan Smith KK7DS <dsmith@danplanet.com>\n"  \
-          "Maurizio Andreotti IZ2LXI <maurizio.iz2lxi@gmail.com>\n" \
+          "Maurizio Andreotti IZ2LXI <maurizioandreotti lc@gmail.com>\n" \
```
Comment (wb8tyw) [inline d_rats/version.py:62]: Just noticed that there is a space in the e-mail address, which makes it invalid.

**Comment by wb8tyw** [comment]
I have created a local branch based on this PR re-based to the current master for testing

**Comment by wb8tyw** [comment]
Just a note here while I am working on a different version of this PR.

We can simplify a lot of the D-Rats code by using the python-pycountry debian module, with similar RPM module or the msys2 mingw-w64-python-pycountry module.

It allows D-Rats to just look at what message catalogs are available instead of us having to maintain a table of language codes and language names.

**Pull request closed without merging**